src/components/EventCard/SearchResultEventCard.tsx

```diff
@@ -91,49 +113,25 @@ const SearchResultEventCard = ({
             <p className="truncate text-sm font-normal">{summary}</p>
           </div>
           <div id="activity-info" className="flex flex-col gap-2">
-            <div className="flex justify-start gap-4">
-              <CalendarDays className="flex-shrink-0" size={24} />
-              <p className="h-6 w-16 flex-shrink-0 text-base font-normal">
-                活動時間
-              </p>
-              <p className="flex-grow truncate text-base font-medium">
-                {startTime}-{endTime}
-              </p>
-            </div>
-            <div className="flex justify-start gap-4">
-              <Users className="flex-shrink-0" size={24} />
-              <p className="h-6 w-16 flex-shrink-0 text-base font-normal">
-                參加人數
-              </p>
-              <p className="flex-grow truncate text-base font-medium">
-                {attendeeCount}
-              </p>
-            </div>
-            <div className="flex justify-start gap-4">
-              <MapPin className="flex-shrink-0" size={24} />
-              <p className="h-6 w-16 flex-shrink-0 text-base font-normal">
-                舉辦位置
-              </p>
-              <p className="flex-grow truncate text-base font-medium">
-                {location}
-              </p>
-            </div>
-            <div className="flex justify-start gap-4">
-              <Building2 className="flex-shrink-0" size={24} />
-              <p className="h-6 w-16 flex-shrink-0 text-base font-normal">
-                主辦單位
-              </p>
-              <p className="flex-grow truncate text-base font-medium">
-                {organizer}
-              </p>
-            </div>
+            <EventCardInfoSection
+              startTime={format(new Date(startTime), 'MM.dd EE ')}
+              endTime={format(new Date(endTime), ' MM.dd EE')}
```

Comment:
我發現問題出在這一段，因為拿到的props已經是轉換過的時間了

```suggestion
              startTime={startTime}
              endTime={endTime}
```

error: 
<img width="996" alt="Screenshot 2024-07-07 at 5 47 41 PM" src="https://github.com/ChillKa/chillka-frontend/assets/70035368/d75c2846-2237-418d-a06f-d569fdec39f8">

Reply:
> 我發現問題出在這一段，因為拿到的props已經是轉換過的時間了
> 
> error: <img alt="Screenshot 2024-07-07 at 5 47 41 PM" width="996" src="https://private-user-images.githubusercontent.com/70035368/346325308-d75c2846-2237-418d-a06f-d569fdec39f8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjAzNDYxMjksIm5iZiI6MTcyMDM0NTgyOSwicGF0aCI6Ii83MDAzNTM2OC8zNDYzMjUzMDgtZDc1YzI4NDYtMjIzNy00MThkLWEwNmYtZDU2OWZkZWMzOWY4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MDclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzA3VDA5NTAyOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU2NzMyMTExYzJkZDU0NDE4ZGNjOTMxYjQyNGVmNjIzOTkwZWQ1ZjljMzk5YmJhNTA4ZjU5Y2Y2NzY2NDU4NTgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.lJiOEuaIEB_r6PTswwxS9Om2oKCg-SwLGRHdgGwSmmw">

我上傳了一個新的commit，看一下可不可以